### PR TITLE
feat: add aged away label

### LINKED ISSUE #852: 🚀 Feature: Add a 'status: aged away' label
### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/create-typescript-app/issues?q=is%3Aissue) and found none that matched my issue.

### Overview

This repository is complex/old enough that I'm starting to see issues that have "aged away" 😄. The first is #799: it no longer reproduces.

A label like `status: aged away` is pretty common for repositories to have. Let's add one in!

### Additional Info

Labels used in this repository are stored in https://github.com/JoshuaKGoldberg/create-typescript-app/blob/a386c870f5bc53ce3b30a8d31108c5726487ba78/src/steps/initializeGitHubRepository/labels/outcomeLabels.ts. You'l want to add a label to that file. See #635 for a previous change like this.

### PR DESCRIPTION
**Pull request by jaas666**
## PR Checklist

- [X] Addresses an existing open issue: fixes #852
- [X] That issue was marked as [`status: accepting prs`](https://github.com/JoshuaKGoldberg/create-typescript-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22status%3A+accepting+prs%22)
- [X] Steps in [CONTRIBUTING.md](https://github.com/JoshuaKGoldberg/create-typescript-app/blob/main/.github/CONTRIBUTING.md) were taken

## Overview

Added new `status: aged away` label

### CODE REVIEW

File: src/steps/initializeGitHubRepository/labels/outcomeLabels.ts [hunk 1]
```diff
@@ -1,84 +1,89 @@
...
-		color: "d876e3",
-		description: "Further information is requested",
-		name: "type: question",
-	},
-	{
-		color: "fde282",
-		description: "Tech debt or other code/repository cleanups",
-		name: "type: cleanup",
-	},
+  {
```
Comment (JoshuaKGoldberg) [inline src/steps/initializeGitHubRepository/labels/outcomeLabels.ts:3]: > ```diff
> -  
> +	
> ```

Hmm, looks like whatever formatter is being applied on the files isn't matching this repository's [Prettier](https://prettier.io) settings. Which is why the diff view of these files is so big - all the indentation changed from `\t` tabs to `  ` two spaces.

https://github.com/JoshuaKGoldberg/create-typescript-app/blob/bfcd795a077a9e4409dce9f99569a5325456bae2/.github/DEVELOPMENT.md should have instructions for getting set up locally. Let me know if any of that's unclear / doesn't work on your end!

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
Functionally looks good, thanks! Just requesting changes on running the formatter.

**Comment by jaas666** [comment]
I think I know what's the issue. I'll fix it later today.

File: src/steps/initializeGitHubRepository/labels/outcomeLabels.ts (hunk 1 above)
Comment (jaas666) [reply src/steps/initializeGitHubRepository/labels/outcomeLabels.ts]: Should be better now.

**Review by JoshuaKGoldberg — APPROVED** [review]
Super, thanks! 🚀 

I'm going to nitpick this on my end and alphabetize by the `name` property 😄 but don't mind me, it's excellent either way.

**Comment by JoshuaKGoldberg** [comment]
@all-contributors please add @jaas666 for code.

> 🤖 Beep boop! This comment was added automatically by [all-contributors-auto-action](https://github.com/marketplace/actions/all-contributors-auto-action).
> Not all contributions can be detected from Git & GitHub alone. Please comment any missing contribution types this bot missed.
> ...and of course, thank you for contributing! 💙

**Comment by jaas666** [comment]
> Super, thanks! 🚀
> 
> I'm going to nitpick this on my end and alphabetize by the `name` property 😄 but don't mind me, it's excellent either way.

Ahhh I should have thought of that.. Thanks

**Pull request merged**